examples/tcpsynbl.yaml

```diff
@@ -0,0 +1,30 @@
+programs:
+  - name: tcpsynbl
+    metrics:
+      histograms:
+        - name: tcp_syn_backlog
+          help: trace the TCP SYN backlog size
+          table: bucket
+          bucket_type: linear
+          bucket_min: 0
+          bucket_max: 1024 # `sysctl net.ipv4.tcp_max_syn_backlog`
+          bucket_multiplier: 1
+          labels:
+            - name: bucket
+              size: 8
+              decoders:
+                - name: uint
+    kprobes:
+      tcp_v4_syn_recv_sock: do_count
+      tcp_v6_syn_recv_sock: do_count
+    code: |
+      #include <net/sock.h>
+
+      // Histograms to record TCP SYN backlog size
+      BPF_HISTOGRAM(bucket, u64);
+
+      int do_count(struct pt_regs *ctx, struct sock *sk) {
+          u64 backlog = bpf_log2l(sk->sk_ack_backlog);
```

Comment:
`sk_ack_backlog` shows the `current listen backlog`

https://www.kernel.org/doc/html/latest/networking/kapi.html?highlight=backlog

Reply:
By the way, you can check [`sk_max_ack_backlog`](https://www.kernel.org/doc/html/latest/networking/kapi.html?highlight=backlog) at the same time and check whether we hit the limit or not.